src/Microsoft.Azure.SignalR/HubHost/ServiceLifetimeManager.cs

```diff
@@ -88,24 +89,21 @@ public override async Task SendConnectionAsync(string connectionId, string metho
 
         private ServiceMessage CreateMessage(string connectionId, string methodName, object[] args, ClientConnectionContext serviceConnectionContext)
         {
+            IDictionary<string, ReadOnlyMemory<byte>> payloads = new Dictionary<string, ReadOnlyMemory<byte>>();
             if (serviceConnectionContext.Protocol != null)
             {
-                var message = new ConnectionDataMessage(connectionId, SerializeProtocol(serviceConnectionContext.Protocol, methodName, args)).WithTracingId();
-                if (message.TracingId != null)
-                {
-                    MessageLog.StartToSendMessageToConnection(Logger, message);
-                }
-                return message;
+                payloads.Add(serviceConnectionContext.Protocol, SerializeProtocol(serviceConnectionContext.Protocol, methodName, args));       
```

Comment:
move the init in L92 into here? Also add a comment describing why we use `MultiConnectionDataMessage`?